server/src/test/java/org/elasticsearch/index/mapper/flattened/FlattenedFieldMapperTests.java

```diff
@@ -423,6 +423,14 @@ public void testDynamicTemplateAndDottedPaths() throws IOException {
         assertEquals(new BytesRef("d\0value"), keyed[0].binaryValue());
     }
 
+    public void testAggregationsDocValuesDisabled() throws IOException {
```

Comment:
Should this be a default implementation in `MapperTestCase` and you override some function to `assumeFalse` it if you can't disable doc values? Or, like, it could catch the exception and assume false from there.